templatestore/views.py

```diff
@@ -8,15 +8,13 @@
 import logging
 import requests
 from templatestore.models import Template, TemplateVersion, SubTemplate, TemplateConfig
-from templatestore.utils import base64decode, base64encode
+from templatestore.utils import base64decode, base64encode, generateDate, generatePayload
 from templatestore import app_settings as ts_settings
 
 logger = logging.getLogger(__name__)
 PDF_URL = ts_settings.WKPDFGEN_SERVICE_URL
 PDF_ASSET_URL = ts_settings.WKPDFGEN_ASSET_URL
-
-
-
+api="https://tinyurl.internal.ackodev.com/api/v1/create_tiny_url"
```

Comment:
This is env specific URL, so it needs to be added to secrets file

Reply:
But we don't push secrets.js file right?
shall i put it in utils file?